Traceback (most recent call last):
  File "/home/filippo/miniconda3/envs/sta2003/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/filippo/miniconda3/envs/sta2003/lib/python3.10/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/filippo/miniconda3/envs/sta2003/lib/python3.10/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
  File "/home/filippo/miniconda3/envs/sta2003/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/home/filippo/miniconda3/envs/sta2003/lib/python3.10/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/home/filippo/miniconda3/envs/sta2003/lib/python3.10/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/filippo/miniconda3/envs/sta2003/lib/python3.10/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
ts_full = PredLoader().get_data('ElecRome')

# Resample the time series to hourly frequency
ts_hourly = np.mean(ts_full.reshape(-1, 6), axis=1)

# Use only the first 3000 time steps
time_series = ts_hourly[0:3000]
time_steps = np.arange(0, len(time_series))
------------------

----- stdout -----
Failed to download ElecRome dataset.
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mAttributeError[0m                            Traceback (most recent call last)
Cell [0;32mIn[4], line 4[0m
[1;32m      1[0m ts_full [38;5;241m=[39m PredLoader()[38;5;241m.[39mget_data([38;5;124m'[39m[38;5;124mElecRome[39m[38;5;124m'[39m)
[1;32m      3[0m [38;5;66;03m# Resample the time series to hourly frequency[39;00m
[0;32m----> 4[0m ts_hourly [38;5;241m=[39m np[38;5;241m.[39mmean([43mts_full[49m[38;5;241;43m.[39;49m[43mreshape[49m([38;5;241m-[39m[38;5;241m1[39m, [38;5;241m6[39m), axis[38;5;241m=[39m[38;5;241m1[39m)
[1;32m      6[0m [38;5;66;03m# Use only the first 3000 time steps[39;00m
[1;32m      7[0m time_series [38;5;241m=[39m ts_hourly[[38;5;241m0[39m:[38;5;241m3000[39m]

[0;31mAttributeError[0m: 'NoneType' object has no attribute 'reshape'

